Header header
uint32  RadomeDamping
uint8   FarCF
uint8   NearCF

